nemo/collections/asr/data/audio_to_label_dataset.py

```diff
@@ -78,21 +78,41 @@ def get_tarred_classification_label_dataset(
     Returns:
         An instance of TarredAudioToClassificationLabelDataset.
     """
-    dataset = audio_to_label.TarredAudioToClassificationLabelDataset(
-        audio_tar_filepaths=config['tarred_audio_filepaths'],
-        manifest_filepath=config['manifest_filepath'],
-        labels=config['labels'],
-        featurizer=featurizer,
-        shuffle_n=shuffle_n,
-        max_duration=config.get('max_duration', None),
-        min_duration=config.get('min_duration', None),
-        trim=config.get('trim_silence', False),
-        shard_strategy=config.get('tarred_shard_strategy', 'scatter'),
-        global_rank=global_rank,
-        world_size=world_size,
-        is_regression_task=config.get('is_regression_task', False),
-    )
-    return dataset
+    tarred_audio_filepaths = config['tarred_audio_filepaths']
+    manifest_filepaths = config['manifest_filepath']
+    datasets = []
+    tarred_audio_filepaths = convert_to_config_list(tarred_audio_filepaths)
+    manifest_filepaths = convert_to_config_list(manifest_filepaths)
+
+    if len(manifest_filepaths) != len(tarred_audio_filepaths):
```

Comment:
Would you please add the support for weighting here?
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/data/audio_to_text_dataset.py#L165

here is a PR example which added weighting for another model:
https://github.com/NVIDIA/NeMo/pull/4530

Reply:
I added support for weighted bucketing